compiler/rustc_error_codes/src/error_codes/E0311.md

```diff
@@ -0,0 +1,69 @@
+This error occurs when there is an unsatisfied outlives bound on a generic
```

Comment:
```suggestion
This error occurs when there is an unsatisfied outlives bound involving an elided region on a generic
```
i think that's the difference between E0309 and E0311, maybe we should merge these two error codes 🤷

Reply:
Thanks for the extra detail here - I updated the text.